roles/filetree_create/README.md

```diff
@@ -206,6 +207,40 @@ A playbook to convert from the structured output to the flattened one is provide
 ansible-playbook infra.controller_configuration.flatten_filetree_create_output.yaml -e '{filetree_create_output_dir: /tmp/filetree_output}'
 ```
 
+## Example Playbook - export object with modifications
+This example will export all object but some with modifications:
+ - job template called `job_template_example` will be exported with the `dev` branch, while the rest of the job templates will use the `main` branch — the resources dictionary takes precedence over the global dictionary.
+ - all projects will have a Jinja2 expression assigned to the `scm_branch`.
+ - all schedules enabled state will be set as `false`.
+```yaml
+---
+- hosts: all
+  connection: local
+  gather_facts: false
+  vars:
+    controller_username: "{{ vault_controller_username | default(lookup('env', 'CONTROLLER_USERNAME')) }}"
+    controller_oauthtoken : "{{ vault_controller_password | default(lookup('env', 'CONTROLLER_OAUTHTOKEN')) }}"
+    controller_hostname: "{{ vault_controller_hostname | default(lookup('env', 'CONTROLLER_HOST')) }}"
+    controller_validate_certs: "{{ vault_controller_validate_certs | default(lookup('env', 'CONTROLLER_VERIFY_SSL')) }}"
+
+    templates_overrides_resources:
+      job_template:
+        job_template_example:
+          scm_branch: "dev"
+
+    templates_overrides_global:
+      job_template:
+        scm_branch: "main"
+      project:
+        scm_branch: !unsafe  "{{ 'true' if AAP.environment == 'PROD' else 'false' }}"
+      schedules:
+        enabled: false
+
+  roles:
+    - infra.controller_configuration.filetree_create
```

Comment:
```suggestion
    - infra. aap_configuration_extended.filetree_create
```

Reply:
I'm not sure if this change has been implemented or not (I'm on holiday and writing from telephone). In the proposal, I believe there's an extra space.